tests/code_update.py

```diff
@@ -51,6 +51,180 @@ def test_verify_quotes(network, args):
 
     return network
 
+@reqs.description("Node with correct security policy joins successfully")
+def test_add_node_with_correct_security_policy(network, args):
```

Comment:
I think this is the "sunny day" scenario that's already tested in the CI when a new node joins successfully on SNP (e.g. `reconfiguration_test_cft`), so this only tests the new `GET /gov/security_policy` endpoint, which I think can be folded into the existing `test_update_all_nodes` test.

Reply:
That's fair although maybe that test then becomes a bit unwieldly? Since joining is tested in the setup maybe I keep this test but just check the policy is what we expect?